tests/test_views.py

```diff
@@ -616,3 +616,113 @@ def test_credit_balance_hidden_when_disabled(self, request_, project, settings):
 
         assert "Credit Balance" not in content
         assert "fa-coins" not in content
+
+
+class TestProjectCreditTransactionsView:
```

Comment:
Best to follow the pattern of other view tests and use the `LoginRequiredMixin` and a `get_url` method.